src/screens/details/Evolution.js

```diff
@@ -1,17 +1,69 @@
 import React from 'react';
-import { SafeAreaView, StyleSheet, Text } from 'react-native';
+import { SafeAreaView, StyleSheet, Dimensions, View, ScrollView } from 'react-native';
 import { useTheme } from 'react-native-paper';
 
 import { DetailsAppBar } from '../../components';
+import EvolutionArrow from '../../components/EvolutionArrow';
+import EvolutionCard from '../../components/EvolutionCard';
 
 export default function Evolution({ color, sprite, chain }) {
   const colors = useTheme().colors;
 
+  var tier1 = chain;
+  var tier2 = chain.evolvesTo;
+  var tier3;
+
+  if (tier2.length == 1) {
+    tier3 = tier2[0].evolvesTo;
+  } else {
+    tier3 = [];
+  }
```

Comment:
You can write it in one line: `var tier3 = tier2.length == 1 ? tier2[0].evolvesTo : [];`